pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3275,6 +3302,9 @@ type CephBlockPoolRadosNamespaceSpec struct {
 	// the CephBlockPool CR.
 	// +kubebuilder:validation:XValidation:message="blockPoolName is immutable",rule="self == oldSelf"
 	BlockPoolName string `json:"blockPoolName"`
+	// Mirroring configuration of CephBlockPoolRadosNamespaceSpec
+	// +optional
+	Mirroring RadosNamespaceMirroring `json:"mirroring,omitempty"`
```

Comment:
@travisn @parth-gr We could also make this a pointer and remove the enable/disable from RadosNamespaceMirroring, WDYT?

Reply:
I just kept how it was with blockpool, if we agree I can change this later with a refactor PR
Or just for rados-namespace for this PR, technically both are possible

Reply:
Should we have a followup or do it in this PR itself?
I am not sure if we can change it for blockpool directly, it would mean that people already using rook would have to refactor as it will be a breaking change (we are going to change the API)

Reply:
Rook does more often use the pattern of having a disabled flag, so maybe we just keep this approach. Let's at least not try to change the blockpool since it would be a breaking schema change